geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLTypes.scala

```diff
@@ -41,7 +41,9 @@ object SQLTypes {
   UDTRegistration.register(classOf[Geometry].getCanonicalName, classOf[GeometryUDT].getCanonicalName)
 
   def init(sqlContext: SQLContext): Unit = {
+    SQLGeometricCastFunctions.registerFunctions(sqlContext)
```

Comment:
@Afterglow375, let's huddle up sometime today and figure out which approach we prefer.